### ISSUE TITLE
Using scan instead of keys

### ISSUE THREAD

**Issue by andy-ganchrow** [joeferner/redis-commander#249]
Because the 'keys' command is blocking, should 'scan' be considered?

https://redis.io/commands/scan

I put together the function below as an example of usage.  This can be used in place of "redisConnection.keys"

```javascript
function scanKeys(keyQuery, cb, cursor = 0, keyAccumulation = []) {
    redisConnection.scan(keyQuery, cursor, (err, result) => {
      if (err) {
        cb(err);
      } else {
        cursor = Number(result[0]);
        let keys = result[1];
        keyAccumulation = keyAccumulation.concat(keys);
        if (cursor === 0) {
          cb(null, keyAccumulation);
        } else {
          scanKeys(keyQuery, cb, cursor, keyAccumulation);
        }   
      }   
    }); 
  }
```

**Comment by styfle** [comment]
It looks like this was fixed in #250

**Comment by sseide** [comment]
The feature requested is available with latest version. Start redis commander with `--use-scan` cli param, set `USE_SCAN` env var or set this param inside your config file (`redis.useScan`)

If there is something missing at the implementation feel free to reopen issue with an updated description.
